### ISSUE TITLE
Wrong cpu percentage output with processes module

### ISSUE THREAD

**Issue by willemdh** [NagiosEnterprises/ncpa#437]
Hello,

When I try NCPA 2.1.3 and use the processes module, I get:

```
/usr/local/nagios/libexec/check_ncpa.py -H nbdig -t 'key' -P 5693 -M 'processes' -w 250 -c 400
OK: Process count was 176 | 'process_count'=176;250;400; 'cpu'=0.565795454545%;;; 'memory'=0.363579545455%;;; 'memory_vms'=6.11GB;;; 'memory_rss'=5.39GB;;;
Processes Matched
PID: Name: Username: Exe: Memory: CPU
-----------------------------------
0: System Idle Process: NT AUTHORITY\SYSTEM: 0.0 % (VMS 0.00 GB, RSS 0.00 GB): 77.38 %
4: System: NT AUTHORITY\SYSTEM: 0.04 % (VMS 0.00 GB, RSS 0.00 GB): 9.97 %
188: mfemms.exe: NT AUTHORITY\SYSTEM: 0.03 % (VMS 0.00 GB, RSS 0.00 GB): 0.00 %
328: svchost.exe: NT AUTHORITY\LOCAL SERVICE: 0.17 % (VMS 0.02 GB, RSS 0.01 GB): 0.00 %

....

15120: chrome.exe: myuser: 1.48 % (VMS 0.12 GB, RSS 0.13 GB): 0.63 %
15324: conhost.exe: myuser: 0.08 % (VMS 0.00 GB, RSS 0.01 GB): 0.10 %

Total Memory: 0.36 % (VMS 6.11 GB, RSS 5.39 GB)
Total CPU: 0.57 %
```

Left about 150 proccess out. So total CPU 0,57 % seems not right here?

Grtz

Willem

**Comment by jomann09** [comment]
Not actually 100% certain why I did this, maybe it was just overlooked, but it's actually an average of the CPU usage per process. Now that I think about it, it probably should just be the added up amount of CPU/Memory usage. Here is the spot https://github.com/NagiosEnterprises/ncpa/blob/master/agent/listener/processes.py#L413 that the issue is. I can fix this for the next version.

**Comment by jomann09** [comment]
Fixed this for 2.1.4 in 9b2243d18a662f9f26d80302bf7c3deb46eddcfb
